pep_sphinx_extensions/pep_zero_generator/writer.py

```diff
@@ -176,6 +187,12 @@ def write_pep0(
             )
             self.emit_newline()
 
+        # PEPs by number
+        if is_pep0:
+            self.emit_title("Numerical Index")
+            self.emit_text("All PEPs :ref:`sorted by number <numerical-index>`.")
```

Comment:
The current text feels too sentence fragment-y. Drawback of the below is two links, though.
```suggestion
            self.emit_text(
                "The :doc:`numerical index </numerical>` contains "
                "a table of all PEPs :ref:`sorted by number <numerical-index>`."
            )
```